src/main/java/net/ltxprogrammer/changed/client/renderer/LatexMantaRayFemaleRenderer.java

```diff
@@ -22,7 +22,9 @@ public LatexMantaRayFemaleRenderer(EntityRendererProvider.Context context) {
                 AbdomenArmor::useAbdomenModel, AbdomenArmor::useInnerAbdomenModel, 0.5f);
         this.addLayer(new LatexParticlesLayer<>(this, this.model));
         this.addLayer(new CustomEyesLayer<>(this, context.getModelSet(),
-                CustomEyesLayer.fixedColor(Color3.fromInt(0xd2d9e1)), CustomEyesLayer.fixedColor(Color3.fromInt(0xffe64b))));
+                CustomEyesLayer.fixedColor(Color3.fromInt(0xd2d9e1)),
+                CustomEyesLayer.fixedColor(Color3.fromInt(0xffe64b)),
```

Comment:
this constructor will call `CustomEyesLayer(scleraColor, leftIrisColor, rightIrisColor)`. You should add a copy of the iris color, then specify eyebrows to noRender, then eyelash color.